docs/how-to-guides/installing-uninstalling-extensions.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Installing and Uninstalling Rancher Desktop Extensions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsConstants from '@site/core/TabsConstants';
+
+Rancher Desktop introduces **Extensions** with the new `v1.9.0-tech-preview` release. The implementation allows for users to extend Rancher Desktop's functionality to better suit their needs. This guide will demonstrate how to install and uninstall extensions on the application.
```

Comment:
```suggestion
The **Extensions** feature introduced in the `v1.9.0-tech-preview` release enables you to use **Docker Desktop Extensions** within Rancher Desktop. The feature helps you extend Rancher Desktop's functionality to meet your additional needs. This guide will demonstrate how to install and uninstall Docker extensions in Rancher Desktop.
```

Reply:
Just updated the intro with new commit, thanks for the review Guna!